.github/CODEOWNERS

```diff
@@ -87,16 +87,20 @@
 #Docs Engineering
 /src/components @aws-amplify/documentation-team
 /src/constants @aws-amplify/documentation-team 
+/src/data @aws-amplify/documentation-team
 /src/directory @aws-amplify/documentation-team 
 /src/pages/_app.tsx @aws-amplify/documentation-team 
 /src/pages/_document.tsx @aws-amplify/documentation-team 
-/src/pages/404.tsx @aws-amplify/documentation-team 
-/src/pages/ChooseFilterPage.tsx @aws-amplify/documentation-team 
+/src/pages/404.tsx @aws-amplify/documentation-team  
+/src/pages/contribute @aws-amplify/documentation-team
 /src/pages/index.tsx @aws-amplify/documentation-team
+/src/pages/**/index.tsx @aws-amplify/documentation-team
 /src/plugins @aws-amplify/documentation-team 
 /src/public @aws-amplify/documentation-team 
-/src/styles @aws-amplify/documentation-team 
+/src/styles @aws-amplify/documentation-team
+/src/themes @aws-amplify/documentation-team 
 /src/utils @aws-amplify/documentation-team
+/tasks @aws-amplify/documentation-team
```

Comment:
I don't know if this is necessarily needed because of the `*` rule at the top of the file?

Reply:
I don't think it's necessary because any files that don't fall under another rule will be owned by @aws-amplify/documentation-team, but it doesn't hurt.